shotover-proxy/tests/transforms/log_to_file.rs

```diff
@@ -20,20 +20,14 @@ async fn log_to_file() {
         "*4\r\n$6\r\nCLIENT\r\n$7\r\nSETINFO\r\n$8\r\nLIB-NAME\r\n$8\r\nredis-rs\r\n",
     );
     let response = std::fs::read("message-log/1/responses/message1.bin").unwrap();
-    assert_eq_string(
-        &response,
-        "-ERR Unknown subcommand or wrong number of arguments for 'SETINFO'. Try CLIENT HELP\r\n",
-    );
+    assert_eq_string(&response, "+OK\r\n");
```

Comment:
swapping to valkey (or a recent version of redis would have worked to) has caused these commands to succeed now since they are now supported by the valkey node.